Helpers/FunctionHelper.js

```diff
@@ -85,6 +85,7 @@ exampleDictionary =
     setconfig: "Use this function to set the configurations for your server. This is an Admin-Only command. Default configurations for your server are provided\
     without the need for this command. This command will let you fine tune this experience in a variety of categories. These categories are:\n\n\
     'Minimum Games' - The minimum number of games before a player shows up in !top\n\
+    'Maximum Top' - The maximum number of players to show up on !top\n\
```

Comment:
"'Leaderboard Length' - The number of players that show up on the leaderboard (!top)\n\"